src/utils/helper.ts

```diff
@@ -1,3 +1,7 @@
-export function generateId(tokenId: string, nftAddress: string): string {
-  return nftAddress + '-' + tokenId
+export function generateId(tokenId: string, tokenAddress: string): string {
```

Comment:
Since we now have roleId and nftId 

```suggestion
export function generateNftId(tokenId: string, tokenAddress: string): string {
```